src/modules/launcher/Plugins/Microsoft.Plugin.Program/Main.cs

```diff
@@ -87,8 +87,11 @@ public List<Result> Query(Query query)
                 .Select(p => p.Result(query.Search, _context.API));
 
             var result = results1.Concat(results2).Where(r => r != null && r.Score > 0);
-            var maxScore = result.Max(x => x.Score);
-            result = result.Where(x => x.Score > Settings.MinScoreThreshold * maxScore);
+            if (result.Any())
+            {
+                var maxScore = result.Max(x => x.Score);
+                result = result.Where(x => x.Score > Settings.MinScoreThreshold * maxScore);
+            }
```

Comment:
Nice.  You might want to call out this issue as a separate PR to be clear what it fixes.    I'm also wondering if it might fix #5658.   Although I believe this code block is handled by an exception.   At the very least you should call this change out in the comments.

Reply:
@ryanbodrug-microsoft Sure! I will put a separate PR for this. 

Also, I don't think this is what caused #5658. This issue was introduced due to a recent change in #5743. All error from plugins are handled in calling code, so it shouldn't be showing up an error window as indicated in #5658

Reply:
PR for this fix: https://github.com/microsoft/PowerToys/pull/6054